### ISSUE TITLE
OAI PMH for repositories that identify resource files in a record element broken

### ISSUE THREAD

**Issue by mjordan** [MarcusBarnes/mik#255]
Running this toolchain against any OAI endpoint results in the following error:

```
DOMDocument::loadXML(): Start tag expected, '<' not found in Entity, line: 2","line":"/home/mark/Documents/hacking/mik/src/writers/Oaipmh.php"
```

**Comment by mjordan** [comment]
@MarcusBarnes I'm working with @bondjimbond on a migration to Arca and thought I'd add a post-write hook script to this branch that transforms the harvested OAI DC into MODs. One trick I used in that script that I think is new is that I instantiate the OAI writer class so I can reuse a method that it defines. If you're interested see https://github.com/MarcusBarnes/mik/blob/issue-255/extras/scripts/postwritehooks/oai_dc_to_mods.php#L42. Reusable code FTW.

@bondjimbond I think all we need to do is refine the XSLT used in this post-write script to output the kind of MODS you're looking for. Everything else appears to be in place to use this toolchain in your migration.

**Comment by mjordan** [comment]
@MarcusBarnes you probably don't need to test the changes I've made to the OAI toolchain. @bondjimbond and I will be running it frequently over the next little while, and the changes don't affect other MIK code. OK if I merge into master if we're happy with the changes on this end?

**Comment by MarcusBarnes** [comment]
@mjordan @bondjimbond - yes, that would be great.  Thank you for contributing this new toolchain.

**Comment by mjordan** [comment]
Great, will get @bonjimbond's feedback from the latest changes and then merge.

**Comment by MarcusBarnes** [comment]
Please also update any documentation as appropriate.

**Comment by mjordan** [comment]
Already done!

**Comment by mjordan** [comment]
RandomSet and SpecificSet fetcher manipulators work out of the box. Neato.

**Comment by MarcusBarnes** [comment]
Addressed with commit 7fed3f0.
